site/docs/encoding/sort.md

```diff
@@ -55,21 +55,22 @@ If the channel has a discrete scale (`band`, `point` or `ordinal`), the field's
 
 3) Unsorted – `null` – The field is not sorted. This is equivalent to specifying `sort: false` in [Vega's scales](https://vega.github.io/vega/docs/scales/#sort).
 
-4) Specify custom order by:
-  1) providing `sort` with an array that lists valid input domains
-  2) providing custom `scale`'s [`domain`](scale.html#domain)
+4) Specify custom order by providing custom `scale`'s [`domain`](scale.html#domain). (In this case, you don't need to use `sort` property.)
```

Comment:
As mentioned earlier, this shouldn't be a "suggested" way for sorting.   Plus, putting unpreferred way on top of preferred way is weird. 

Please just make this like a note that it can achieve similar effect but has some pitfall(s).    (The `sort` array can be 4. and this should become just a note.)

Actually, we should also mention about `sort` in scale's `domain` documentation too.

Reply:
I think we already have that in `domain` documentation (https://vega.github.io/vega-lite/docs/scale.html#domain)

Reply:
Yes, but it didn't say what's the different between domain array and sort array yet so briefly mention about it and link to the full example in sort page would be nice.